CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
-
+* PayPal
+  * Add PayPal App Switch flow (BETA)
```

Comment:
Can we add a couple notes on how to use the feature here as well? We can use iOS as a reference: https://github.com/braintree/braintree_ios/blob/main/CHANGELOG.md#6210-2024-06-12

Reply:
Sure, updated: [4914007](https://github.com/braintree/braintree_android/pull/1166/commits/4914007efc5333ccf4927bb281b2ac96664a92f5) let me know if this looks good